File: js/bee1041.js
var input = require('fs').readFileSync('js/stdin', 'utf8')
var [x,y] = input.split(' ').map(Number)
if (x === 0 && y === 0) {
  console.log('Origem')
} else {
  if (x === 0) {
    console.log('Eixo Y')
  } else if (y === 0) {
    console.log('Eixo X')
  } else if (x>0 && y>0) {
    console.log('Q1')
  } else if (x<0 && y>0) {
    console.log('Q2')
  } else if (x<0 && y<0) {
    console.log('Q3')
  } else if (x>0 && y<0) {
     console.log('Q4')
  }
}

File: C/bee1041.c
#include <stdio.h>
#include <string.h>

int main(){
    double x,y;
    char quadrante[100] = "";
    scanf("%lf %lf", &x, &y);
    if (x == 0 && y == 0){
        strcat (quadrante, "Origem");    
    } else if(x == 0){
        strcat (quadrante, "Eixo Y");
    }else if(y == 0){
        strcat (quadrante, "Eixo X");
    }else if(x > 0 && y > 0){
        strcat (quadrante, "Q1");
    }else if(x < 0 && y > 0){
        strcat (quadrante, "Q2");
    }else if(x < 0 && y < 0){
        strcat (quadrante, "Q3");
    }else if(x > 0 && y < 0){
        strcat (quadrante, "Q4");
    }
    printf("%s\n", quadrante);
}